src/crypto/CryptoInterface.c

```diff
@@ -174,6 +196,24 @@ int BSL_Crypto_WrapKey(BSL_Data_t *wrapped_key, BSL_Data_t cek, const char *kek_
     }
     wrapped_key->len += (size_t)final_len;
```

Comment:
Similar to above I think. The length cannot just be added to without actually reallocing the buffer, but I think this just happens to work if `final_len` is zero.